version1/02 - Object types.md

```diff
@@ -90,863 +92,614 @@ The following notation is used in the tables below:
-|           | ref       | refNasjonalIdentifikator         |          |             |                    |  |      |      |   x   |      |  x   |        |                                                                                                                                                               |
-|           | ref       | refKryssreferanseTilMappe        |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktMappe** that the current **AbstraktRegistrering** cross references                                                                        |
-|           | ref       | refKryssreferanseFraMappe        |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktMappe** that cross reference the current **AbstraktRegistrering**                                                                         |
-|           | ref       | refKryssreferanseTilRegistrering |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that the current **AbstraktRegistrering** cross references                                                                 |
-|           | ref       | refKryssreferanseFraRegistrering |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that cross reference the current **AbstraktRegistrering**                                                                  |
-
-### Journalpost
-
-| Field no. | Type      | Field                             | Not null | Has default |    Code list       |  | Save | View | Query | Sort | Link | Unlink | Comment                                                                                                                                                                                                                                                                                                                                                                                                         |
+Note: **EFO** required if finalized
+
+Note: **EJO** required if journaled
+| Field no. | Type      | Field                            | Not null |   Default   | Code list          |  | Save | View | Query | Sort | Link | Unlink | Comment |
+|:----------|:----------|:---------------------------------|:--------:|:-----------:|:-------------------|:-|:----:|:----:|:-----:|:----:|:----:|:------:|:--------|
+| M110      | date      | offentlighetsvurdertDato         |          |             |                    |  |  V   |  V   |   V   |  V   |      |        |         |
```

Comment:
This field actually belongs to the base type (basisregistrering in this documentation) and has to be moved.

Arkivnotat has support for `refArkivnotatDokumentflyt` that needs to be listed here, instead.